src/backend/scripts/setup_localdata.py

```diff
@@ -328,18 +329,20 @@ def create_test_data(engine):
     _ = create_gisaid(session)
 
     # Create db rows for our main test user
-    group = create_test_group(session, "CZI", "CZI")
-    user = create_test_user(session, group, "User1", "Test User")
     location = create_location(
         session, "North America", "USA", "California", "San Mateo County"
     )
+    group = create_test_group(session, "CZI", "CZI", location)
+    user = create_test_user(session, group, "User1", "Test User")
```

Comment:
Set up a default tree build location for the groups we create in local development to make it easier to test generating nextstrain configs.